microbench/src/main/java/alluxio/inode/RocksBenchBase.java

```diff
@@ -0,0 +1,161 @@
+import org.rocksdb.RocksDBException;
+import org.rocksdb.WriteOptions;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class RocksBenchBase {
+
+  static final String SER_READ = "serRead";
+  static final String NO_SER_READ = "noSerRead";
+  static final String SER_NO_ALLOC_READ = "serNoAllocRead";
+  static final String NO_SER_NO_ALLOC_READ = "noSerNoAllocRead";
+
+  private final RocksInodeStore mRocksInodeStore;
+  private final RocksDB mDB;
+  private final AtomicReference<ColumnFamilyHandle> mInodesColumn;
+  private final WriteOptions mDisableWAL;
+
+  RocksBenchBase(String confType) throws IOException {
+    Logger.getRootLogger().setLevel(Level.ERROR);
+    String dir =
+        AlluxioTestDirectory.createTemporaryDirectory("inode-store-bench").getAbsolutePath();
+    RocksBenchConfig.setRocksConfig(confType, dir, Configuration.modifiableGlobal());
+    mRocksInodeStore = new RocksInodeStore(dir);
+    Pair<RocksDB, AtomicReference<ColumnFamilyHandle>> dbInfo = mRocksInodeStore.getDBInodeColumn();
+    mDB = dbInfo.getFirst();
+    mInodesColumn = dbInfo.getSecond();
+    mDisableWAL = new WriteOptions().setDisableWAL(true);
+  }
+
+  static MutableInode<?> genInode(boolean mIsDirectory) {
+    if (mIsDirectory) {
+      return MutableInodeFile.create(0, 1, "testFile", System.currentTimeMillis(),
+          CreateFileContext.defaults());
+    } else {
+      return MutableInodeDirectory.create(0, -1, "someParent",
+          CreateDirectoryContext.defaults());
+    }
+  }
+
+  void after() {
+    mRocksInodeStore.clear();
+    mRocksInodeStore.close();
+  }
+
+  long getInodeReadId(long totalCount, long nxtId, int threadCount, int threadId) {
+    // since the larger ids were written last (and will be in the memtable), but the zipfan will
+    // generate small numbers the most often, we subtract from the total count to read the
+    // large ids most often
+    // this assumes a workload where latest writes are read more often
+    return (totalCount - ((nxtId * threadCount) + threadId) % totalCount) - 1;
+  }
+
+  Inode readInode(long totalCount, long nxtId, int threadCount, int threadId) {
+    return mRocksInodeStore.get(getInodeReadId(totalCount, nxtId, threadCount, threadId)).get();
+  }
+
+  Inode readInode(long totalCount, long nxtId, int threadCount, int threadId, byte[] inodeBytes) {
+    return get(getInodeReadId(totalCount, nxtId, threadCount, threadId),
+        inodeBytes).map(Inode::wrap).get();
+  }
+
+  byte[] readInodeBytes(long totalCount, long nxtId, int threadCount, int threadId) {
+    return getBytes(getInodeReadId(totalCount, nxtId, threadCount, threadId)).get();
+  }
+
+  int readInodeBytes(long totalCount, long nxtId, int threadCount, int threadId,
+                     byte[] inodeBytes) {
+    return getBytes(getInodeReadId(totalCount, nxtId, threadCount, threadId), inodeBytes);
+  }
```

Comment:
```suggestion
  long getInodeReadId(long totalCount, long nxtId) {
    // since the larger ids were written last (and will be in the memtable), but the zipfan will
    // generate small numbers the most often, we subtract from the total count to read the
    // large ids most often
    // this assumes a workload where latest writes are read more often
    return (totalCount - nxtId % totalCount) - 1;
  }

  Inode readInode(long totalCount, long nxtId) {
    return mRocksInodeStore.get(getInodeReadId(totalCount, nxtId)).get();
  }

  Inode readInode(long totalCount, long nxtId, byte[] inodeBytes) {
    return get(getInodeReadId(totalCount, nxtId), inodeBytes).map(Inode::wrap).get();
  }

  byte[] readInodeBytes(long totalCount, long nxtId) {
    return getBytes(getInodeReadId(totalCount, nxtId)).get();
  }

  int readInodeBytes(long totalCount, long nxtId, byte[] inodeBytes) {
    return getBytes(getInodeReadId(totalCount, nxtId), inodeBytes);
  }
```